.github/scripts/uitests_for_ios.sh

```diff
@@ -54,13 +55,19 @@ for file in $(find integration_test -maxdepth 1 -type f); do
   zip -r "ios_tests.zip" "Release-iphoneos" "Runner_iphoneos$dev_target-arm64-armv7.xctestrun"
   popd
 
+#  echo "--------------------------------------------------------"
```

Comment:
Should we keep this commented code?

Reply:
yes, I kept it for local use of the script